pydata_sphinx_theme/__init__.py

```diff
@@ -529,7 +532,7 @@ def setup(app):
 
     # Update templates for sidebar
     pkgdir = os.path.abspath(os.path.dirname(__file__))
-    path_templates = os.path.join(pkgdir, "_templates")
+    path_templates = os.path.join(pkgdir, "templates")
```

Comment:
Flag for @jorisvandenbossche, any reason we wouldn’t want to use templates wo the underscore? I don’t think this would affect anything downstream

Reply:
It won't affect theme users. When theme users override a template, it's based on the file's basename, not on its path within a particular theme. The current layout actually makes it possible to accidentally have the same basename in pydata_sphinx_theme and pydata_sphinx_theme/templates, such that one of the files is shadowed by the other.